frontend/app/dashboard/src/views/dashboard/settings/modules/members/ImportMembersView.vue

```diff
@@ -108,396 +128,378 @@
+import { Address, Organization, OrganizationPatch, RecordAddressAnswer, RecordDateAnswer, RecordSettings, RecordTextAnswer, RecordType } from "@stamhoofd/structures";
 import XLSX from "xlsx";
-
 import { allMatchers } from "../../../../../classes/import/defaultMatchers";
 import { ImportingMember } from "../../../../../classes/import/ImportingMember";
 import { MatchedColumn } from "../../../../../classes/import/MatchedColumn";
 import { MatcherCategory } from '../../../../../classes/import/MatcherCategory';
-import { AddressColumnMatcher, DateColumnMatcher,TextColumnMatcher } from "../../../../../classes/import/matchers";
+import { AddressColumnMatcher, DateColumnMatcher, TextColumnMatcher } from "../../../../../classes/import/matchers";
 import ImportMembersErrorsView from './ImportMembersErrorsView.vue';
 import ImportMembersQuestionsView from './ImportMembersQuestionsView.vue';
 import ImportVerifyProbablyEqualView from './ImportVerifyProbablyEqualView.vue';
-
-@Component({
-    components: {
-        STNavigationBar,
-        STToolbar,
-        STInputBox,
-        STErrorsDefault,
-        Checkbox,
-        BackButton,
-        LoadingButton,
-        Dropdown
-    },
-})
-export default class ImportMembersView extends Mixins(NavigationMixin) {
-    errorBox: ErrorBox | null = null
-    validator = new Validator()
-    saving = false
-    file: null | string = null
-    organizationPatch: AutoEncoderPatchType<Organization> & AutoEncoder = OrganizationPatch.create({})
-
-    rowCount = 0
-    columnCount = 0
-
-    sheets: Record<string, XLSX.WorkSheet> = {}
-    internalSheetKey: string | null = null
-
-    matchers = allMatchers.slice()
-    columns: MatchedColumn[] = []
-
-    created() {
-        this.organizationPatch.id = this.$organization.id
+import { onMounted, ref, computed, Ref } from 'vue'
+import { useLegacyMemberManager } from '@stamhoofd/registration';
+
+const organization = useOrganization();
+const memberManager = useLegacyMemberManager();
+const show = useShow();
+const errorBox: Ref<ErrorBox | null> = ref(null);
+const saving = ref(false);
+const file = ref<string | null>(null);
+const organizationPatch: AutoEncoderPatchType<Organization> & AutoEncoder = OrganizationPatch.create({})
+
+let rowCount = 0
+let columnCount = 0
+
+const sheets = ref<Record<string, XLSX.WorkSheet>>({});
+const internalSheetKey = ref<string | null>(null);
+
+const matchers = ref<typeof allMatchers>(allMatchers.slice());
+const columns = ref<MatchedColumn[]>([]);
+
+//#region created
+/**
+* Because setup is run around the beforeCreate and created lifecycle hooks, you do not need to explicitly define them
+*/
+if (organization.value !== null)
+    organizationPatch.id = organization.value.id;
+//#endregion
+
+const sheetKey = computed<string | null>(() => internalSheetKey.value);
```

Comment:
```suggestion
const sheetKey = internalSheetKey;
```
Dit is identiek, dus denk dat dit weg kan